spell checker/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Spell Checker</title>
+</head>
+<body>
+    <script>
+        var userArray = [];
+        var correctArray = [];
+        function spellChecker(userwords, correctwords){
+            for(var i = 0; i < 10; i++){
+                if(userwords[i] !== correctwords[i]){
+                    userArray.push(userwords[i])
+                    correctArray.push(correctwords[i])
+                }
+                
+            }
+            return "Correct Words: " + correctArray + " And User Words: " + userArray;
```

Comment:
You should be returning an object with two properties, "correct" which has an array of the correct words, and "incorrect" which has an array of the incorrect words